lib/conf.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    identities: {
+        prefix: 'http://localhost:8001/api/v1/',
+        token: 'test IdentityStore'
+    },
+    registrations: {
```

Comment:
@imsickofmaps I don't think we've had a 'create_change' equivalent to date... am I right in presuming that this will be a PATCH HTTP request?